extensions/amp-facebook-comments/amp-facebook-comments.md

```diff
@@ -52,6 +52,12 @@ You can use the `amp-facebook-comments` component to embed the [Facebook comment
 
 The URL of the comments page. For example, `http://www.directlyrics.com/adele-25-complete-album-lyrics-news.html`.
 
+##### data-locale (optional)
+
+By default, the local is set to <code>en_US</code>; however, you can specify a locale as well.
```

Comment:
Default is `user's system language` right?

Reply:
ditto other components

Reply:
Done